pkg/kubectl/cmd/util/printing.go

```diff
@@ -92,16 +92,16 @@ func PrinterForCommand(cmd *cobra.Command) (kubectl.ResourcePrinter, bool, error
 		return nil, generic, err
 	}
 
-	return maybeWrapSortingPrinter(cmd, printer), generic, nil
+	return maybeAddSortingPreprocessor(cmd, printer), generic, nil
 }
 
-func maybeWrapSortingPrinter(cmd *cobra.Command, printer kubectl.ResourcePrinter) kubectl.ResourcePrinter {
+func maybeAddSortingPreprocessor(cmd *cobra.Command, printer kubectl.ResourcePrinter) kubectl.ResourcePrinter {
```

Comment:
Add a TODO to eliminate the flag check here.  Whoever is asking for sorting should pass the field(s) they want to sort on.